src/game/gui/textinput.c

```diff
@@ -27,6 +27,7 @@ typedef struct {
 
     textinput_done_cb done_cb;
     void *userdata;
+    text_object text_cache[3];
```

Comment:
Can be 2 not 3 here

Reply:
These text_caches should seriously be named variables.

Reply:
I don't disagree, or at least used named indices

Reply:
Sounds good.